components/ExerciseCard/ExerciseCard.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+import { NewButton } from '../theme/Button'
+import { Text } from '../theme/Text'
+
+export type ExerciseCardProps = {
+  challengeName: string
+  problem: string
+  answer: string
+  explanation: string
+  className?: string
+}
+
+const ExerciseCard = ({
+  challengeName,
+  problem,
+  answer,
+  explanation,
+  className = ''
```

Comment:
What is the need for this `className` prop?

Reply:
It can be useful for adding CSS classes. For example if you want to add margin-bottom you can pass in "mb-2" `<ExerciseCard className="mb-2" ... />`.

Reply:
Are there any pages in the designs that require this use case?

Reply:
I'm not sure yet. Maybe I can remove the className prop until it's actually needed.

Reply:
Yeah lets remove it for now

Reply:
Done